src/hacktimer.js

```diff
@@ -0,0 +1,93 @@
+let fakeIdToCallback = {};
+
+function uuidv4() {
+  return '10000000-1000-4000-8000-100000000000'.replace(/[018]/g, (c) =>
+    (
+      c ^
+      (crypto.getRandomValues(new Uint8Array(1))[0] & (15 >> (c / 4)))
+    ).toString(16)
+  );
+}
+
+window.setInterval = function (callback, time /* , parameters */) {
```

Comment:
Is it possible to not override window? I am afraid this will override stuff from captcha side too

Reply:
> captcha side too

what do you mean by that?

Reply:
I am afraid that hCaptcha's setInterval may use the setInterval function from the extension. Can it be modified to use export instead, so that it can be imported similar to worker-timers?

Reply:
hCaptcha will not be able to use our setInterval in any way since our script is executed in an [isolated environment](https://developer.chrome.com/docs/extensions/mv3/content_scripts/#isolated_world)
if you want me to change to export and import, I can do it, I just didn't see the point in it

Reply:
ah okay did not know that thanks!